kafka-minion/templates/deployment.yaml

```diff
@@ -19,9 +19,11 @@ spec:
         {{- with .Values.podAnnotations }}
         {{- toYaml . | nindent 8 }}
         {{- end }}
+        {{- if eq .Values.serviceMonitor.create false }}
```

Comment:
You can use `if not` here instead I think?

Reply:
makes sense